code/modules/cm_tech/tech.dm

```diff
@@ -58,6 +58,9 @@
 
 	return TRUE
 
+/datum/tech/proc/is_tier_tech()
+	return FALSE
```

Comment:
This ought to just be a var on the tech https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#getters-and-setters